Xamarin/DrawXShared/DrawXSettingsManager.cs

```diff
@@ -60,20 +41,43 @@ public static DrawXSettings Settings
                 {
                     Write(() =>
                     {
-                        _savedSettings = new DrawXSettings { LastColorUsed = "Indigo" };
-                        _localSettingsRealm.Add(_savedSettings);
+                        _savedSettings = _localSettingsRealm.Add(new DrawXSettings
+                        {
+                            LastColorUsed = "Indigo"
+                        });
                     });
                 }
 
                 return _savedSettings;
             }
         }
 
+        public static User CurrentUser
+        {
+            get
+            {
+                try
+                {
+                    return User.Current;
+                }
+                catch
+                {
+                    // We have ended up with more than one logged in user. Log all of them out.
```

Comment:
I don't like this any more than I did the other solution. How can we accidentally end up with more than one logged in user? That's a bug and we shouldn't conceal it..